python/lsst/cp/pipe/cpDark.py

```diff
@@ -114,19 +124,36 @@ def run(self, inputExp):
         # exposure, if possible.  Otherwise, use the cameraGeom value.
         gains = self._get_gains(inputExp)
 
+        # Is this gainContext still required?  TODO DM-48754:
+        # Investigate cosmic ray rejection during dark construction
         with gainContext(inputExp, inputExp.getVariance(), apply=True, gains=gains):
             # Scale the variance to match the image plane.  A similar
             # scaling happens during flat-field correction for science
             # images.
             self.log.debug("Median image and variance: %f %f",
                            np.median(inputExp.image.array), np.median(inputExp.variance.array))
+            crImage = inputExp.clone()
+            # Interpolate the crImage, so the CR code can ignore
+            # defects (which will now be interpolated).
+            interpolateFromMask(
+                maskedImage=crImage.getMaskedImage(),
+                fwhm=self.config.psdfFwhm,
+                growSaturatedFootprints=self.config.crGrow,
+                maskNameList=list(self.config.maskListToInterpolate),
+                useLegacyInterp=self.config.useLegacyInterp,
+            )
+
             try:
-                self.repair.run(inputExp, keepCRs=False)
+                self.repair.run(crImage, keepCRs=False)
             except LengthError:
                 self.log.warning("CR rejection failed!")
+
+            # Copy results to input frame.
+            crBit = crImage.mask.getPlaneBitMask('CR')
```

Comment:
This file uses double-quotes not single (for the most part...).

Reply:
I've switched to double quotes.  They have slightly different meanings in perl, and I can't unlearn ~three decades of habit.